modules/ROOT/pages/6.0-release-notes-commands-and-queries.adoc

```diff
@@ -0,0 +1,78 @@
+= TinyMCE 6.0 Commands and queries changes
+:navtitle: TinyMCE 6.0 Commands and queries changes
+:description: TinyMCE 6.0 Commands and queries changes
+:keywords: releasenotes, commands, queries
+
+This section documents the improvements and changes made to {productname} 6.0 Commands and queries.
+
+It also documents the {productname} 6.0 Commands that have been removed.
+
+
+* xref:commands-and-queries[Commands and queries]
+* xref:new-and-improved-commands-and-queries[New and improved commands and queries]
+* xref:changed-commands-and-queries[Changed commands and queries]
+* xref:removed-or-deprecated-commands-and-queries[Removed or deprecated commands and queries]
+
+
+// tag::commands-and-queries[]
+[[commands-and-queries]]
+== Commands and queries
+
+
+[[new-and-improved-commands-and-queries]]
+=== New and improved commands and queries
+
+
+[[mce-autocompleter-close-and-mce-autocompleter-reload]]
+==== `mceAutocompleterClose` and `mceAutocompleterReload`
+
+As part of moving `autocompleter` from the default `silver` theme to the {productname} core, two new commands are now available: `mceAutocompleterClose` and `mceAutocompleterReload`.
```

Comment:
```suggestion
As part of moving core autocompleter feature from the default `silver` theme to the {productname} core, two new commands are now available: `mceAutocompleterClose` and `mceAutocompleterReload`.
```

Reply:
Also do we need to mention what these commands do?

Reply:
Deferring Lee's second comment